### ISSUE TITLE
Problem with spacing in return value

### ISSUE THREAD

**Issue by Ran5m** [FreeCodeCamp/FreeCodeCamp#3577]
Challenge [Waypoint: Iterate over Arrays with map](http://freecodecamp.com/challenges/waypoint-iterate-over-arrays-with-map) has an issue.
User Agent is: <code>Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_5) AppleWebKit/601.1.56 (KHTML, like Gecko) Version/9.0 Safari/601.1.56</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

My code:
```javascript
//Use map to add three to each value in the array
var array = [1,2,3,4,5];
// Only change code below this line.
array = array.map (function(val) {
  return val+3;
});
// Only change code above this line.
(function() {return array;})();

```
If you include spaces in the return statement (for example ```return val + 3;``` as opposed to ```return val+3``` as in the code above, the waypoint fails with an error indicating the map function hasn't been used.

**Comment by ahstro** [comment]
From what I could see, the problem was actually with having a space between `.map` and `(`. I'm submitting a PR in a second that fixes this.

**Comment by ahstro** [comment]
f

**Comment by ltegman** [comment]
This was fixed by PR #3613 and should be closed

**Comment by bugron** [comment]
@Ran5m thanks for posting this issue!
Closing this issue as https://github.com/FreeCodeCamp/FreeCodeCamp/pull/3613 is merged and fixes this issue. Thanks, @ltegman.

>***
>Please review the [**Guidelines for Contributing**](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md), thank you!.
>***
